chevah/brink
# [#3112] Publish on staging under different username.

Scope
=====

This updated brink publish code to work with our new staging server.

Changes
======

allow different username for staging.... and no longer have hardcoded usernamed


How to test
=======

reviewers: @alibotean 

check that changes make sense.

I will use it to publish our latest branch for AIX fix on staging.
